docs/source/index.rst

```diff
@@ -53,6 +53,11 @@ Tried and tested models for diseases, risks, and interventions from the 2017 Glo
 
 Tried and tested models for disease, risks, and interventions from the 2019 Global Burden of Disease Study.
 
+:ref:`Existing Models for GBD 2019 <2020_models>`
```

Comment:
@Ninicorn Looks like this should say "GBD 2020" not "GBD 2019". And regarding the failing checks:

One of the doc build warnings says that `2020_models` is an undefined label, and another one says the file `docs/source/gbd2020_models/risk_exposures/stunting/index.rst` is not included in any toctree.

Maybe try merging the latest version of `main` into your branch and see if that fixes the failing checks?